Assessment.Test/Assessment.Test/App.xaml.cs

```diff
@@ -12,9 +14,9 @@ public partial class App : Application
         public App()
         {
             InitializeComponent();
-            Page page = Activator.CreateInstance(typeof(HomeView)) as Page;
-            page.BindingContext = DependencyService.Get<HomeViewModel>();
-            MainPage = page;
+            Page page = Activator.CreateInstance(typeof(LoginView)) as Page;
+            page.BindingContext = DependencyService.Get<LoginViewModel>();
+            MainPage = new NavigationPage(new LoginView());
```

Comment:
Don't create a content page instance directly in here. Assign the page variable Main Page.use page variable to viewmodel binding and instance crations